bokehjs/src/coffee/palettes/palettes.coffee

```diff
@@ -454,8 +454,91 @@ palettes = {
                    0xd7e219, 0xdae218, 0xdce218, 0xdfe318, 0xe1e318, 0xe4e318, 0xe7e419, 0xe9e419, 0xece41a, 0xeee51b, 0xf1e51c, 0xf3e51e,
                    0xf6e61f, 0xf8e621, 0xfae622, 0xfde724]
   }
+
+  Accent: {
+    Accent3     : [0x7fc97f, 0xbeaed4, 0xfdc086]
+    Accent4     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99]
+    Accent5     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0]
+    Accent6     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f]
+    Accent7     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f, 0xbf5b17]
+    Accent8     : [0x7fc97f, 0xbeaed4, 0xfdc086, 0xffff99, 0x386cb0, 0xf0027f, 0xbf5b17, 0x666666]
+  }
+
+  Dark: {
```

Comment:
friendly ping @stvn66 will you be able to make this last minor change? I would be nice to include this in `0.12.1` if possible, but that's in the next few days.